tsconfig.json

```diff
@@ -1,9 +1,10 @@
 {
   "compilerOptions": {
     "baseUrl": "src",
-    "target": "es6",
+    "target": "esnext",
```

Comment:
Is there a reason for changing this?  Will it cause us issues, as those things part of ESNext are proposals, and not always fully supported, right?